src/app/containers/FeaturesAnalysis/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, node } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import { C_GHOST } from '@bbc/psammead-styles/colours';
+import styled from 'styled-components';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import featuresAnalysis from '#pages/StoryPage/featuresAnalysis.json';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedLarge } from '#lib/styledGrid';
+import StoryPromo from '../StoryPromo';
+
+const Wrapper = styled(GridItemConstrainedLarge)`
+  margin-bottom: ${GEL_SPACING_DBL};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+  /* z-index needs explicitly set as the psammead section label component uses negative z-indices */
+  z-index: 0;
+`;
+
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0;
+
+  span {
+    background-color: transparent;
+  }
+
+  span[dir] {
+    background-color: ${C_GHOST};
+  }
+`;
+
+const FeaturesAnalysis = ({ content }) => {
+  const { script, service, dir } = useContext(ServiceContext);
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': 'features-analysis-heading',
+  };
+  const FeaturesAnalysisWrapper = ({ children }) => (
+    <Wrapper {...a11yAttributes}>{children}</Wrapper>
+  );
```

Comment:
Why does this warrant a new component? Can we attached `a11yAttributes` on line 56?

Reply:
This style was borrowed from `CpsRelatedContent` container.

In that case, the relevant code is:

```jsx
const RelatedContentWrapper = ({ children }) =>
    enableGridWrapper ? (
      <GhostGrid {...a11yAttributes}>
        <Wrapper>{children}</Wrapper>
      </GhostGrid>
    ) : (
      <Wrapper {...a11yAttributes}>{children}</Wrapper>
    );
```

It's clear that we need it to decide between wrapping with `<GhostGrid />` or not, but it's not clear why `<Wrapper />`, which the `RelatedContentWrapper` styles, is still included in its descendants. 

Perhaps, this has some effect on a11y I don't know about?